.travis.yml

```diff
@@ -47,5 +52,12 @@ script:
       else
         mvn org.jacoco:jacoco-maven-plugin:prepare-agent install org.jacoco:jacoco-maven-plugin:report sonar:sonar;
       fi
+
+    - cp target/serverUpgradeBundle.zip docker/
+    - cd docker
+    - unzip serverUpgradeBundle.zip
+    - docker build -f Dockerfile-osc-base . -t osc-base-image:osc-base
```

Comment:
can we just execute `createDockerImage` ant target here? we are pretty much doing the same thing here manually